### PR TITLE
TICK-371 Adds an example comparing tick and scikit-learn for L1 logistic regression

### PR DESCRIPTION
**Pull request by stephanegaiffas**

### CODE REVIEW

**Comment by stephanegaiffas** [comment]
@Mbompr : so do you approve ?

File: examples/plot_logistic_tick_vs_scikit.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+from sklearn.linear_model import LogisticRegression as LogRegScikit
+
+from tick.dataset import fetch_tick_dataset
+from tick.inference import LogisticRegression as LogRegTick
+
+train_set = fetch_tick_dataset('binary/adult/adult.trn.bz2')
+test_set = fetch_tick_dataset('binary/adult/adult.tst.bz2')
+
+clf_tick = LogRegTick(C=1e5, penalty='l1')
+clf_scikit = LogRegScikit(penalty='l1')
```
Comment (Mbompr) [inline examples/plot_logistic_tick_vs_scikit.py]: Do you think we should add a shared tol parameter?

Comment (stephanegaiffas) [inline examples/plot_logistic_tick_vs_scikit.py]: Yes I'll do it.

File: examples/plot_logistic_tick_vs_scikit.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+t1 = time()
+clf_tick.fit(train_set[0], train_set[1])
+t_tick = time() - t1
```
Comment (Mbompr) [inline examples/plot_logistic_tick_vs_scikit.py]: for performance measurement it's better to use time.clock()
http://pythoncentral.io/measure-time-in-python-time-time-vs-time-clock/

**Pull request merged**